packages/contracts/src/resources/site.ts

```diff
@@ -530,12 +548,19 @@ export const siteApi = contract.router(
 		},
 	},
 	{
+		strictStatusCodes: true,
 		pathPrefix: "/api/v0/c/:communitySlug/site",
 		baseHeaders: z.object({
 			authorization: z
 				.string()
 				.regex(/^Bearer /)
 				.optional(),
 		}),
+		commonResponses: {
+			// this makes sure that 400 is always a valid response code
+			400: zodErrorSchema,
+			403: z.string(),
+			404: z.string(),
+		},
```

Comment:
did not know this was an option, now every route has the correct error responses at least.